### ISSUE TITLE
Include ability to add input to Step Function Execution (and have it returned in describe_execution)

### ISSUE THREAD

**Issue by ciaranevans** [spulec/moto#3283]
**TL;DR** Moto should actually handle `input` for the Execution model [here](https://github.com/spulec/moto/blob/master/moto/stepfunctions/models.py#L29-L47)

@bblommers I intend on implementing this, just raising the issue so I can A: link the PR to something and B: not forget I need to do this 🤣 

I'm planning on adding `input` as a property of `Execution` and implementing this back into [describe_execution](https://github.com/spulec/moto/blob/master/moto/stepfunctions/responses.py#L126-L141)

This is so that when a user runs `describe_execution` against a moto Step Function, they get something like [this](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/stepfunctions.html#SFN.Client.describe_execution):
```
{
    'executionArn': 'string',
    'stateMachineArn': 'string',
    'name': 'string',
    'status': 'RUNNING'|'SUCCEEDED'|'FAILED'|'TIMED_OUT'|'ABORTED',
    'startDate': datetime(2015, 1, 1),
    'stopDate': datetime(2015, 1, 1),
    'input': 'string',
    'output': 'string'
}
```

**Comment by bblommers** [comment]
This is now part of moto >= 1.3.15.dev1047
